resources/public/api/conf/1.0/application.yaml

```diff
@@ -0,0 +1,1981 @@
+openapi: 3.0.3
+info:
+  title: CTC Traders API
+  contact: {}
+  version: '1.0'
+  description: |
+    # Overview
+    Use the CTC Traders API to send departure and arrival movement to the New Computerised Transit System (NCTS). 
+    Retrieve messages sent from customs offices of departure and destination.
+servers:
+  - url: https://test-api.service.hmrc.gov.uk
+    description: Sandbox
+  - url: https://api.service.hmrc.gov.uk
+    description: Production
+    variables: {}
+paths:
+  /customs/transits/movements/arrivals:
+    post:
+      tags:
+      - common-transit-convention-traders
+      summary: Send an arrival notification message
+      description: >
+        Notify the Customs Office of Destination that a transit movement has arrived at its final destination.
+
+        This request is also called an ‘Arrival Notification’ E_ARR_NOT (IE007) message.
+
+        A successful response will contain a URI and an arrival ID for the notification, which you can use to retrieve the message status.
+
+        It will also contain an embedded request ID, box ID and box name, which you can use to match this movement to any messages received from the Push Pull Notifications API.
+
+        **Note:** If an Economic Operators Registration and Identification (EORI) number has not been assigned to you, an HTTP 403 Forbidden error is returned.
+      operationId: SendAnArrivalNotificationMessage

```

Comment:
See guidelines for operation ids -  https://swagger.io/docs/specification/paths-and-operations/#operationid

Although there's leeway for how you write an `operationId`, I would be inclined to trim the word count of each `id`  and use HTTP keywords (`get`, `post`, `patch`, `put`, `delete`).

Suggestions:
postArrivalNotificationMessage